convert/jenkinsjson/convert.go

```diff
@@ -636,6 +636,32 @@ func collectStepsWithID(currentNode jenkinsjson.Node, stepWithIDList *[]StepWith
 	case "jiraSendDeploymentInfo":
 		*stepWithIDList = append(*stepWithIDList, StepWithID{Step: jenkinsjson.ConvertJiraDeploymentInfo(currentNode, variables), ID: id})
 
+	case "s3Upload":
```

Comment:
Add unit tests for those rules.

Reply:
@Fernando-Dourado I have written unit test cases for all three new APIs, Extract Entries, ConvertS3Archieve, ConvertsS3Upload . tested trace json with single and multiple entries for the test cases as well. didn't get what are rules here?

Reply:
We have code coverage missing on convert.go file. As you added rules inside the s3Upload case, we need to cover them.